engine/config/ConEntry.cpp

```diff
@@ -1,7 +1,8 @@
 #include "ConEntry.h"
 
+#include <core/Engine.h>
```

Comment:
I leave includes of engine headers after system and std library headers

lmk if that's a bad idea because I've just been rolling with it since day 1 and don't know what the recommended header include order is, but that's the project convention at least

Reply:
ah ok. yeah I just put includes wherever as it doesn't seem to really affect anything.